app/src/main/res/values/dimens.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="activity_horizontal_margin">12dp</dimen>
+    <dimen name="activity_vertical_margin">12dp</dimen>
```

Comment:
This affects everything. All activities. Not just editor/viewer.

But let's try. It can be reverted if it is bad.

Reply:
I may have set the margin too small. 😬